easybuild/tools/modules.py

```diff
@@ -59,9 +59,10 @@
 VERSION_ENV_VAR_NAME_PREFIX = "EBVERSION"
 DEVEL_ENV_VAR_NAME_PREFIX = "EBDEVEL"
 
-# keep track of original LD_LIBRARY_PATH, because we can change it by loading modules and break modulecmd
+# keep track of original $LD_LIBRARY_PATH/$LD_PRELOAD, because it can change by loading modules and break module command
 # see e.g., https://bugzilla.redhat.com/show_bug.cgi?id=719785
-LD_LIBRARY_PATH = os.getenv('LD_LIBRARY_PATH', '')
+LD_ENV_VAR_KEYS = ['LD_LIBRARY_PATH', 'LD_PRELOAD']
+ORIG_ENVIRON = dict([(key, os.getenv(key, '')) for key in LD_ENV_VAR_KEYS])
```

Comment:
certainly less than 1000! ;-)

I didn't consider cleaning this up, but now I will. I'll try and get only `easybuild.tools.environment' to keep a copy of the original environment in a (private) global variable, and let the others query it via a dedicated function